File: boot-other/src/main/kotlin/com/hjf/boot/demo/boot/cache/kotlin/ManServiceImpl.kt
package com.hjf.boot.demo.boot.cache.kotlin

import org.springframework.stereotype.Service

/**
 * name: com.hjf.boot.demo.boot.cache.kotlin
 * author: mickjoust
 * date: 2018/5/10
 **/
@Service
class ManServiceImpl : ManService {
    override fun findByName(name: String): Man {
        return Man(1,name,"12323131313")
    }
}

File: boot-other/src/main/scala/com/hjf/boot/demo/boot/cache/scala/ManServiceImpl.scala
package com.hjf.boot.demo.boot.cache.scala

import org.springframework.stereotype.Service

/**
  * name: com.hjf.boot.demo.boot.cache.scala
  * author: mickjoust
  * date: 2018/5/10
  **/
@Service
class ManServiceImpl extends ManService {
  override def findByName(name: String) = new Man(1,name,"test@test")
}
